docs/visualization/index.md

```diff
@@ -21,23 +25,21 @@
 - [create_communication_latency_plot](./visualization_api/communication_information.md#latency)
 - [create_communication_frequency_plot](./visualization_api/communication_information.md#frequency)
 - [create_communication_period_plot](./visualization_api/communication_information.md#period)
-
-#### Node
```

Comment:
@hsgwa @takeshi-iwanari 
I adapted your comment.
> "Node" の下に、"Publish/Subscription Information" があるのが違和感がありました。"Node"を親にするなら"Callback"などもその下に来るべき?
> callbackよりかは、Communicationの下にある方が、実装的には近いです。
> comm.publisher, comm.subscription
> 等へもアクセスできますし

I deleted Node subsection and put `create_publish_subscription_period_plot` function in communication subsection.

Reply:
LGTM